src/wrapper_array.jl

```diff
@@ -181,6 +176,7 @@ end
     return setindex!(a, value, inds...)
 end
 
-@propagate_inbounds function Base.setindex!(a::NamedDimsArray, value, inds...)
+@propagate_inbounds function Base.setindex!(a::NamedDimsArray, value, raw_inds...)
+    inds = Base.to_indices(a, raw_inds)
```

Comment:
I don't believe this is necessary. Can you provide a case where this succeeds and the previous version fails?

Reply:
I agree, I think I unthinkingly made them the same. Will remove.